README.md

```diff
@@ -57,6 +57,36 @@ texts = ["someone talks on the phone angrily while another person sits happily",
          "someone talks on the phone happily while another person sits angrily"]
 scores = clip_flant5_score(images=images, texts=texts) # scores[i][j] is the score between image i and text j
 ```
+## (Preview) NEW! - Video Inference:
+
+This is a preview version of a new feature for T2V: video inference! Note that this feature requires installation of `flash-attn`. Please do this in addition to the evironment setup above to align with your system environment. 
+
+For single-image and CLIP-like models, video frames are concatenated. For all other native interleaved-image/video models (we recommend LLaVA-OneVision at the time of writing), video frames are passed directly to the model.
+
+```python
+import t2v_metrics
+
+### For a single (video, text) pair on a single-image model:
```

Comment:
In addition to image-based VQAScore, I think it will be great to add 1-2 line demos for each of the new models you add (especially the real video VQAScore models)!

Reply:
Sorry, I think the typos made it seem like I was just using CLIP-FlanT5, but I also do a representative example for LLaVA-OV. The others that are multi-image like mPLUG or InternVL2 have the same usage.